src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,117 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private int capacity = 10;
```

Comment:
[Any magic numbers should be constants](https://github.com/LaverAndrii/jv-array-list/blob/hw-array-list/checklist.md#any-magic-numbers-should-be-constants)